frontend-v2/src/features/data/LoadData.tsx

```diff
@@ -1,6 +1,6 @@
 import { Box, Stack, Typography } from "@mui/material";
 import Papa from "papaparse";
-import { FC, useCallback, useState } from "react";
+import { FC, useCallback, useState, useEffect } from "react";
```

Comment:
I think you can remove `useState` and `useEffect` now.
```suggestion
import { FC, useCallback } from "react";
```

Reply:
Correct, missed that - I'm used to eslint blocking the commit with unused vars. Updated